src/client/components/side-drawer/side-drawer.tsx

```diff
@@ -16,7 +16,9 @@ export interface SideDrawerProps extends React.Props<any> {
   onOpenAbout: Fn;
   onClose: Fn;
   customization?: Customization;
-  showOverviewLink?: boolean;
+  isHome?: boolean;
+  isCube?: boolean;
+  isLink?: boolean;
```

Comment:
I think this has to be a single `viewType` parameter instead of 3 booleans. Going to merge anyways.